### PR TITLE
Some issue fixes

### PR DESCRIPTION
**Pull request by Badiboy**
## Description

- Pass default parse mode to Media items:
https://github.com/eternnoir/pyTelegramBotAPI/issues/2232
- Hide token in exception messages:
https://github.com/eternnoir/pyTelegramBotAPI/issues/2227

### CODE REVIEW

**Comment by Badiboy** [comment]
@coder2020official Please, take a look. It's short )

**Comment by coder2020official** [comment]
Is showing token a big bug though? Isn't it necessary for debug purposes, etc?

**Comment by coder2020official** [comment]
If you really want to hide the token - I propose adding some checks for token to contain colon; Then, add a function to mask the letter part of the token(the one after colon); and show this format of token

**Comment by coder2020official** [comment]
So, when creating a TeleBot instance, check for ":" in token; raise error if not present
Then, in any exception, call a function to replace after-colon part with asterisk;

**Comment by Badiboy** [comment]
> Is showing token a big bug though? Isn't it necessary for debug purposes, etc?

Big bug - no. Necessary for debug - also not. 
Token is more or less sencitive thing, thus showing it in logs may be harmful in some cases.

**Comment by Badiboy** [comment]
> If you really want to hide the token - I propose adding some checks for token to contain colon; Then, add a function to mask the letter part of the token(the one after colon); and show this format of token

What the reason to do this processing? What additional value will be given by hiding the letter part instead of replace the token as a whole?

**Comment by coder2020official** [comment]
- Use asterisk (if I understood your question directly)
- The first part shows Bot ID, which might be necessary for debugging

**Comment by Badiboy** [comment]
> * Use asterisk (if I understood your question correctly)
> * The first part shows Bot ID, which might be necessary for debugging

+

**Comment by Badiboy** [comment]
image

@coder2020official Should it be await-ed?

**Comment by coder2020official** [comment]
no

File: telebot/async_telebot.py [hunk 1]
```diff
@@ -367,9 +367,15 @@ async def infinity_polling(self, timeout: Optional[int]=20, skip_pending: Option
...
                 if logger_level and logger_level >= logging.ERROR:
+                    e = str(e)
+                    if self.token in e:
+                        e = e.replace(self.token, 'TOKEN')
                     logger.error("Infinity polling exception: %s", str(e))
                 if logger_level and logger_level >= logging.DEBUG:
-                    logger.error("Exception traceback:\n%s", traceback.format_exc())
+                    e = traceback.format_exc()
+                    if self.token in e:
+                        e = e.replace(self.token, 'TOKEN')
```
Comment (coder2020official) [inline telebot/async_telebot.py:377]: mmm?

File: telebot/async_telebot.py [hunk 2]
```diff
@@ -367,9 +367,15 @@ async def infinity_polling(self, timeout: Optional[int]=20, skip_pending: Option
...
+                    logger.error("Exception traceback:\n%s", e)
```
Comment (coder2020official) [inline telebot/async_telebot.py:378]: Did you forget about this part?

**Review by coder2020official — CHANGES_REQUESTED** [review]

File: telebot/async_telebot.py (hunk 2 above)
Comment (Badiboy) [reply telebot/async_telebot.py:378]: Ups...

File: telebot/__init__.py [hunk 3]
```diff
@@ -8110,11 +8127,17 @@ def register_business_message_handler(self,
...
+
+        :param regexp: Regular expression
+        :type regexp: :obj:`str`
+
         :param func: Function executed as a filter
         :type func: :obj:`function`
 
-        :param pass_bot: True if you need to pass TeleBot instance to handler(useful for separating handlers into different files)
-        :type pass_bot: :obj:`bool`
+        :param content_types: Supported message content types. Must be a list. Defaults to ['text'].
```
Comment (coder2020official) [inline telebot/__init__.py:8139]: does it?...
we are not setting content_types

**Review by coder2020official — APPROVED** [review]
I am not sure on some parts, as I think some parts may not throw token error; Anyways, it is not bad to still have replacing there; So I think it looks good now

File: telebot/__init__.py (hunk 3 above)
Comment (Badiboy) [reply telebot/__init__.py:8139]: image

I just align description with parameters.

Comment (coder2020official) [reply telebot/__init__.py:8139]: Ah ok, seems like we need another full PR with all content_types description fixes + default values. you can merge

**Comment by Badiboy** [comment]
🤝

**Pull request merged**